pom.xml

```diff
@@ -741,7 +761,7 @@ flexible messaging model and an intuitive client API.</description>
         </plugin>
         <plugin>
           <artifactId>maven-compiler-plugin</artifactId>
-          <version>2.3.2</version>
+          <version>3.7.0</version>
```

Comment:
I was seeing the errors in the other PR but wasn't able to figure out why would those test be failing with the newer compiler plugin.

Reply:
oops, i didn't mean to push this. Will remove. Though it would be nice to get the other one in, as I end up changing this a couple of times a day. I rekicked a build on it earlier.